### ISSUE TITLE
Input-Validator

### ISSUE THREAD

**Issue by Mika215** [Mika215/separation-of-concerns-starter#26]
- This is a component that will display an error message(reminder) that the user did not enter a proper input.
- in fact this was previously functional in our to-do list app using the built in alert function. However,since we are suggested to replace it with a different mechanism i am working over it.

**Comment by RosaMoran** [comment]
yes, this issue i wanted speak about it, cuz we haven't any code for input user  and also i want to discute maybe it better if you don't add my part i mean delete part into `inneHTML` cuz that gave me troubles

**Comment by Mika215** [comment]
Hey @RosaMoran,
Regarding adding the delete button in the innerHTML file; i don't really think that i will provide you any solutions.
As you may remember last Sunday i had tried to raise this question in the class and i have clearly mentioned that this was the main problem that is blocking the whole group from progressing ahead.Besides, i have further asked if there is any other mechanism that we can solve this problem.
However,  Bermarte responded that he gave you a solution for that specific problem and that we should have to check the code that he suggested you.
In fact you were sharing your screen and you showed us some sort of code.
Back then i was not understanding clearly and i was hoping that you would explain to us what he has explained to you regarding this specific issue.

The only thing that i remember is that he suggested  you to assign classes to the buttons that you want to target inside the innerHTML and then you can target them using the classes when ever you want to modify(manipulate them).
if i am not mistaken he was saying that this mechanism is called `Delegation`. Reviewing YouTube tutorials or browsing some documents on this topic might help a bit.
I don't think there is a way to take the buttons out of the innerHTML since they should have to be crated dynamically with the dome. unless you will have a static buttons that will never go away whether the user has submitted an activity(to do list) or not.
